### DISCUSSION TITLE
PDF Not Rendering

### DISCUSSION THREAD

**Discussion by abbasmashaddy72** [spatie/browsershot#675] [Bugs]
Controller:
`Browsershot::html('Foo')
            ->setNodeBinary('/usr/bin/node')
            ->setNpmBinary('/usr/bin/npm')
            ->pdf();`

PDF View:
![image](https://user-images.githubusercontent.com/55267488/190574070-07a3d014-39c6-46b0-8523-3d96806d6d4b.png)

Save PDF is working Fine, but I don't want to save the file I want to view it directly.

**Comment by thiagoalmeida88** [comment]
Hi, the secret is in the option: disposition = 'inline'

Change your code to: 

`return response()->streamDownload(function () {
            echo Browsershot::html('Foo')
                ->setNodeBinary('/usr/bin/node')
                ->setNpmBinary('/usr/bin/npm')
                ->pdf();
        }, 'file_name.pdf', ['Content-Type' => 'application/pdf'], 'inline');`

Tell me if it worked.
